runtime/near-vm-runner/src/wasmer2_runner.rs

```diff
@@ -369,11 +369,11 @@ impl Wasmer2Config {
 }
 
 // We use following scheme for the bits forming seed:
-//  kind << 10, kind is 1 for Wasmer
+//  kind << 10, kind is 1 for Wasmer2
 //  major version << 6
 //  minor version
 const WASMER2_CONFIG: Wasmer2Config = Wasmer2Config {
-    seed: (1 << 10) | (2 << 6) | 0,
+    seed: (1 << 10) | (3 << 6) | 0,
```

Comment:
I'm not entirely sure if we want to bump `major` version here or the minor one. Either way this does not actually reflect the version of the VM implementation? so to me it seems like we want to just have a plain counter here just like for Wasmer0.

Reply:
Yeah, it's not super-clear what versioning scheme we should use here. Right now we deliberatlly don't use semver for our version of wasmer, and instead use `=` version in Cargo.toml. I guess, ideally we should mix some constant defined in the wasmer crate itself?